sdk/textanalytics/ai-text-analytics/README.md

```diff
@@ -1,30 +1,12 @@
-# Azure Text Analytics client library for JavaScript
+# Azure Text Analytics Client library for JavaScript
 
-[Azure TextAnalytics](https://azure.microsoft.com/services/cognitive-services/text-analytics/) is a cloud-based service that provides advanced natural language processing over raw text, and includes six main functions:
+This package contains an isomorphic SDK for Azure Text Analytics Client.
```

Comment:
I would add an explanation as to what isomorphic even means here. Or reword it so that we dont use the word at all.

Reply:
Done.